### DISCUSSION TITLE
Add playback settings to the footer

### DISCUSSION THREAD

**Discussion by Hajuskin** [jcorporation/myMPD#435] [Ideas]
I find it a little cumbersome to first go to the playback tab in order to change basic playback controls like random or repeat. I think it would feel more natural to have them near to the other playback controls in the footer. 

Also, integrating the "Local Playback" to the "Output" settings might help to free some space in the footer.

**Comment by jcorporation** [comment]
Is already implemented for myMPD 6.12.0. Playback settings can be accessed in playback and queue card and optional  through a button in the footer.

**Reply by Hajuskin** [reply]
Perfect :)
